Google.Api.Gax.Grpc/ApiCallTracingExtensions.cs

```diff
@@ -48,4 +95,36 @@ internal static class ApiCallTracingExtensions
     internal static Func<CallSettings, AsyncDuplexStreamingCall<TRequest, TResponse>> WithTracing<TRequest, TResponse>(
         this Func<CallSettings, AsyncDuplexStreamingCall<TRequest, TResponse>> fn, ActivitySource activitySource, string methodName) =>
         fn;
+
+    // This is still very much up in the air, and may even require changes to the parameters, so that we get more information
+    // (e.g. the full RPC name, the client name etc).
+    private static string FormatActivityName(Delegate fn, string methodName) => $"{fn.Method.Name}/{methodName}";
+
+    // TODO: See if there's a standard way of doing this. It seems odd to have to do it ourselves.
```

Comment:
I was thinking the same.
This actually looks more like a logging concern than a tracing one? But yes, let's see what others do or if there are best practices etc.

Reply:
I think setting the status to be an error is probably important for tracing though.

Reply:
Yep, the status is relevant, but the exception message and the stack trace seem odd in tracing, at least to me.